lib/engine/game/g_1840/entities.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G1840
+      module Entities
+        MARKET = [
+            %w[40],
+             ].freeze
+
+        PHASES = [
+            {
+              name: 'a',
+              train_limit: 3,
+              tiles: [:yellow],
```

Comment:
can't you do like %i[yellow]

Reply:
this file is just there that I can test the map